new recyclerview for the fields

ManuelKiendl opened the pull request.

ManuelKiendl committed:
new recyclerview for the fields

fatjoe3012 commented:
I scheiße

fatjoe3012 commented:
Is scheiße

fatjoe3012 commented on app/src/main/java/com/example/smarteden/data/Field.kt:
```diff
@@ -0,0 +1,5 @@
+package com.example.smarteden.data
+
+class Field {
+
```
De Klasse gfollt ma richtig guad :D

fatjoe3012 commented on app/src/main/java/com/example/smarteden/greenhousefields/RecyclerViewAdapter.kt:
```diff
@@ -0,0 +1,40 @@
+    val descriptionTextView = itemView.findViewById<TextView>(R.id.textview_description)
+
+}
+class RecyclerViewAdapter : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
```
Da ViewHolder is normal innerhalb der adapter klasse

ManuelKiendl replied:
moansd du mei klasse cardviewholder ? soll de in en recyclerviewadapter ?

fatjoe3012 replied:
Ja

fatjoe3012 replied:
De Namensgebung gfollt ma a no ned. RecyclerViewAdapter für wos? Do wad irgendwie sche, wenn irgendwos dabei steht von Felder oda sowos

ManuelKiendl replied:
ok danke für de detect info des habi mi grod davor gfragt wie i de ganzen sachan oschaun ko

fatjoe3012 commented on app/src/main/res/navigation/nav_graph.xml:
```diff
@@ -3,7 +3,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/nav_graph"
```
Als start fragment hätt i gern wieder as FirstFragment -> Login

ManuelKiendl replied:
Klaro🧔🏿🧔🏿🧔🏿

fatjoe3012 commented on app/src/main/res/values/strings.xml:
```diff
@@ -6,6 +6,8 @@
     <string name="next">Next</string>
     <string name="previous">Previous</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
I hoff den String brauch ma nimma 🙈

ManuelKiendl replied:
wo kummt der kas her

fatjoe3012 commented:
Außerdem wad a Beschreibung vom Commit immer von Vorteil, dass ma woas, wos du drim host.
mit ".\gradlew detekt" konnst im Terminal detekt laffa lassen. Es is immer besser, wenn ma de Fehler vorm Pushen identifizieren kann

fatjoe3012 commented on app/src/main/java/com/example/smarteden/greenhousefields/RecyclerViewAdapter.kt:
```diff
@@ -0,0 +1,40 @@
+    val titleTextView= itemView.findViewById<TextView>(R.id.textview_title)
+    val subtitleTextView = itemView.findViewById<TextView>(R.id.textview_subtitle)
+    val descriptionTextView = itemView.findViewById<TextView>(R.id.textview_description)
+
```
Für wos stehen do eigentlich title, subtitle und description? 🙈 
Und wos für Daten lodsd du do eine? Werd ma wahrscheinlich a Viewmodel mit Live Data braucha

ManuelKiendl replied:
Ja des is ja etz blos aktuell drinnen des werdi no alles verändern nur dasi de basics scho mal drin hab

ManuelKiendl committed:
new recyclerview for the fields

ManuelKiendl committed:
new recyclerview for the fields
*
RecyclerView getestet und angepasst

ManuelKiendl committed:
Recyclerview angepasst
*
Daten werden jetzt über Lifedata / Viewmodel in Recyclerview gebracht
Haut oba noned so ganz hi

ManuelKiendl committed:
Recyclerview angepasst
*
OnClickListener für jede CardView.
Anchließend öffnet sich ein neues Fragment dem die id übergeben wird und wo zukünftig alle aktuellen Daten enthalten sein sollen und evtl auch einstellungen vorgenommen werden können

ManuelKiendl committed:
Neues Fragment Fieldinfo.kt
*
Im neuen fragment_fieldinfo.xml werden die gesamten Daten des aktuellen Feldes angezeigt und es lässt sich stand jetzt die Bewässerung steuern
Evtl werden diese Funktionen aber noch erweitert und man kann dinge wie Licht oder Temperatur steuern

ManuelKiendl committed:
Neues Fragment Fieldinfo.kt
*
Im neuen fragment_fieldinfo.xml werden die gesamten Daten des aktuellen Feldes angezeigt und es lässt sich stand jetzt die Bewässerung steuern
Evtl werden diese Funktionen aber noch erweitert und man kann dinge wie Licht oder Temperatur steuern

ManuelKiendl committed:
Merge branch 'develop' into fieldoverviewtest

ManuelKiendl committed:
Neues Fragment Fieldinfo.kt
*
Im neuen fragment_fieldinfo.xml werden die gesamten Daten des aktuellen Feldes angezeigt und es lässt sich stand jetzt die Bewässerung steuern
Evtl werden diese Funktionen aber noch erweitert und man kann dinge wie Licht oder Temperatur steuern

ManuelKiendl committed:
Merge remote-tracking branch 'origin/fieldoverviewtest' into fieldoverviewtest

ManuelKiendl committed:
Neues Fragment Fieldinfo.kt
*
Im neuen fragment_fieldinfo.xml werden die gesamten Daten des aktuellen Feldes angezeigt und es lässt sich stand jetzt die Bewässerung steuern
Evtl werden diese Funktionen aber noch erweitert und man kann dinge wie Licht oder Temperatur steuern
